src/templates/drizzle/logo.hbs

```diff
@@ -1,6 +1,42 @@
-    </g>
-</svg>
+<div class="sds-logo">
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Layer_1" viewBox="0 0 542.4 154.8">
+        <title>Spark Design System</title>
+        <style>
+            .st1{fill:#FF721E}.st2{fill:#FF8920}.st3{fill:#FFA022}.st4{fill:#FF5C1C}.st5{fill:#FF451A}.st7{fill:#FFFFFF}
+        </style>
+        <defs>
+            <path id="SVGID_1_" d="M113.9 98.2c5.8-2.6 8.6-9.4 6.2-15.1-1.2-2.9-3.5-5.1-6.4-6.3-2.9-1.2-6.1-1.1-9 .2L73.3 91c-13.5 6-29.4.1-35.5-13.2-3-6.5-3.2-13.8-.7-20.6 2.5-6.7 7.6-12.1 14.1-15l51.2-23.4 17.6-8c3.2-1.4 2.6-6.1-.8-6.8-1.5-.3-3.1-.5-4.7-.5h-.1l-57.7.2c-8.5 0-16.4 4.6-20.6 12l-28.7 50c-4.2 7.4-4.2 16.5.1 23.9l24.2 41.6c.9 1.6 2.9 2.2 4.5 1.5l11.1-4.9 66.6-29.6z"/>
+        </defs>
+        <clipPath id="SVGID_2_">
+            <use overflow="visible" xlink:href="#SVGID_1_"/>
+        </clipPath>
+        <g clip-path="url(#SVGID_2_)">
```

Comment:
clip-paths can cause issues if the random generated id someone gets used somewhere else - anyway to create the logo by releasing that clippath in the vector editor and using svg shapes instead?